src/sif/bicyclecost.cc

```diff
@@ -32,6 +32,7 @@ constexpr float kDefaultCountryCrossingPenalty = 0.0f; // Seconds
 constexpr float kDefaultUseRoad = 0.25f;               // Factor between 0 and 1
 constexpr float kDefaultUseFerry = 0.5f;               // Factor between 0 and 1
 constexpr float kDefaultAvoidBadSurfaces = 0.25f;      // Factor between 0 and 1
+const std::string kDefaultBicycleType = "Hybrid";      // Bicycle type
```

Comment:
I added this, but is `Hybrid` the correct default here?

Reply:
yes. I prefer Road but the majority of use cases seem to be city bikes or "hybrid" bikes.

Reply:
yes, it was inconsistent between code and docs. It is now consistent as `Hybrid`

Reply:
It looks like the docs still say `Road` is the default.


https://github.com/valhalla/valhalla-docs/blob/master/turn-by-turn/api-reference.md#costing-models

![pasted_image_7_17_18__4_54_pm](https://user-images.githubusercontent.com/5665333/42844794-1b831b86-89e2-11e8-9220-a31aa8ee2339.png)

Reply:
We moved the docs to here: https://github.com/valhalla/valhalla/blob/master/docs/api/turn-by-turn/api-reference.md#bicycle-costing-options

I will update the old docs too